WordPress/Classes/ViewRelated/NUX/Signin2FAViewController.swift

```diff
@@ -75,6 +76,17 @@ import WordPressShared
     // MARK: Configuration Methods
 
 
+    /// Assigns localized strings to various UIControl defined in the storyboard.
+    ///
+    func localizeControls() {
+        verificationCodeField.placeholder = NSLocalizedString("Verification code", comment: "two factor code placeholder")
+
+        let submitButtonTitle = NSLocalizedString("VERIFY", comment: "Title of a button. The text should be uppercase.").localizedUppercaseString
+        submitButton.setTitle(submitButtonTitle, forState: .Normal)
+        submitButton.setTitle(submitButtonTitle, forState: .Highlighted)
```

Comment:
It is. :) In the past I've seen the titles for highlighted and selected state not update when a new title is assigned to the normal state. I don't recall the exact situation, or if the selected/highlighted titles had been assigned. 
The docs say ``` At a minimum, you should set the value for the normal state. If a title is not specified for a state, the default behavior is to use the title associated with the UIControlStateNormal state ``` so, yah, setting just the normal title should be fine according to the docs.  However, I prefer to be cautious and set the highlighted state as well. Particularly since we're configuring the buttons via a storyboard and a highlighted title set there would end up being a non-obvious bug.